2026/maf.md

```diff
@@ -69,23 +69,59 @@ The final score will be computed over a set of $x$ private instances that are si
 
 ### Input Format
 
-We use the standard [Newick format](https://en.wikipedia.org/wiki/Newick_format) for phylogenetic trees.
-
-Each tree is represented by a valid parenthesis-expression. Each internal vertex $u$ of the tree is represented by the expression $(E_1,E_2,\ldots,E_d)$ where each $E_i$ is an expression for the subtree rooted at the $i$'th child of $u$. Each leaf of the tree is represented by its label. The whole expression is terminated with a semi-colon.
+Each instance is described by a single text file that consists of different line types:
+ - Empty lines carry no strict meaning and can be ignored. 
+ - Lines starting with the symbol `#` and *can* always be ignored, but may contain useful additional data.
+   The meaning of these lines is determined by the second character:
+   - `#p {k} {L}` indicate that the file contains $k$ trees with $L$ leaves each.
+     This line appears in the header as the first non-comment line.
+   - `# ` is a comment line.
+   - We may add further line types in a backward-compatible way later, in the sense that it is never wrong to ignore an unrecognized `#` line.
+ - All other non-empty lines contain a tree description in the Newick format (see below) and represent an input tree.
+   There are exactly $k$ such lines.
```

Comment:
The literature traditionally uses n as the number of leaves, k as the size of the agreement forest (number of trees therein), and t as the number of input trees. I also used this nomenclature in the verbal description of the problem on the webpage. We might want to be consistent there.

Reply:
Hi Mathias, thank you for pointing that out. I misread your comment. It now should be fixed.